src/comps/Calendar.js

```diff
@@ -1,30 +1,24 @@
 import React from "react";
-import { makeStyles } from "@material-ui/core";
+import Box from "@mui/material/Box";
 import FullCalendar from "@fullcalendar/react";
```

Comment:
Also another calendar thing, is it possible to change the colour of the current day in dark mode?
![ThingAN2214](https://user-images.githubusercontent.com/90653148/192077668-3316f3bb-2a7e-43f9-b237-eee2f7e66983.png)

Reply:
Yup! Fullcalendar takes another CSS variable to customize the coloration of the today highlight, and I have a theme-aware use of it in 89473539597d831bdeb649ebd22b2549d54d3a82. Let me know what you think about the current color I set (`rgba(255, 255, 255, 0.05)`).